src/storage/proto/wal.proto

```diff
@@ -12,14 +12,9 @@ message WalHeader {
 enum PayloadType {
   NONE = 0;
   WRITE_BATCH_ARROW = 1;
-  WRITE_BATCH_PROTO = 2;
```

Comment:
Do we still need PayloadType?

Reply:
Looks like not necessary to have the payload type as we can use mutation_types to check whether the payload is empty.

Reply:
Looks like not necessary to have the payload type as we can use mutation_types to check whether the payload is empty.